### PR TITLE
added additional endpoints

### PR DESCRIPTION
**Pull request by alexonab**

### CODE REVIEW

**Comment by knurts** [comment]
This looks awesome, can't wait to test it out

File: webull.py [hunk 1]
```diff
@@ -300,6 +302,83 @@ def get_quote(self, stock='') :
...
+            direction: all, calls, puts
+            expireDate: contract expire date
+            queryAll: 0
+        '''
+        if not expireDate:
+            expireDate = self.get_options_expiration_dates(stock)[0]['date']
+        url = f'https://quoteapi.webullbroker.com/api/quote/option/{self.get_ticker(stock)}/list'
+        params = {'count': count, 'includeWeekly': includeWeekly, 'direction': direction,
+            'unSymbol': stock, 'queryAll': queryAll}
+        return requests.get(url, params=params).json()['data']
```
Comment (knurts) [inline webull.py]: I don't know if 'data' is a valid key here. You can use the following to figure out what it should be.
try:
    return requests.get(url, params=params).json()['data']
except KeyError as e:
    print(requests.get(url, params=params).json())
    print("Key error. Bad key: {}".format(e))
    return False

Comment (alexonab) [reply webull.py]: Data is a valid key for the endpoint that is being called.  See the example below.
`
>>> from webull import webull
>>> webull = webull()
>>> webull.get_bars('TSLA', interval='m1', extendTrading=1) 
                       open    high     low   close  volume   vwap
timestamp                                                         
2020-01-15 08:11:00  532.55  532.55  532.28  532.28   122.0  538.0
`

**Comment by tedchou12** [comment]
@alexonab Thanks Mike!
@knurts Thank you for the cleanup!

**Pull request merged**